zcash/client.go

```diff
@@ -36,11 +36,9 @@ var defaultPeerConfig = &peer.Config{
 	//
 	// If this version is too low, newer peers will disconnect from the DNS seeder,
 	// and it will only be able to talk to outdated peers.
-	//
-	// TODO: fork https://github.com/gtank/btcd/blob/master/peer/peer.go
-	//       and set MinAcceptableProtocolVersion based on the most recently activated network upgrade
-	//       see ticket #10 for details
 	ProtocolVersion: 170100, // Zcash NU5 mainnet
+	// TODO: ideally we would set 170100 for Mainnet, but there's no mechanism for that right now
+	MinAcceptableProtocolVersion: 170050, // Zcash NU5 testnet
```

Comment:
I'm not sure this change achieves the goal of this PR, we'll still have some outdated nodes on the mainnet seeder. (Or on the testnet seeder between NU6 testnet activation and mainnet activation.)

Can we make the minimum protocol version part of the per-network ChainParams config instead?

That would need changes to this PR and https://github.com/ZcashFoundation/btcd/pull/2

Reply:
I did this in 063fef4. I didn't change `btcd` because it seems it wouldn't make any difference to put it in `ChainParams` instead of `Config`